Plugin~/WebRTCPluginTest/GraphicsDeviceTestBase.cpp

```diff
@@ -0,0 +1,77 @@
+﻿#include "pch.h"
+#include "GraphicsDeviceTestBase.h"
+#include "../WebRTCPlugin/GraphicsDevice/GraphicsDevice.h"
+
+using namespace WebRTC;
+
+#if defined(_WIN32)
```

Comment:
#if defined(SUPPORT_D3D11) is probably better ?

Reply:
fixed it